src/Raven.Server/Integrations/PostgreSQL/Messages/MessageBuilder.cs

```diff
@@ -374,7 +374,7 @@ private void WriteByte(byte value, ref int pos)
 
         private void VerifyBufferSize(int pos, int addedSize)
         {
-            if (pos + addedSize > Buffer.Length)
+            while (pos + addedSize > Buffer.Length)
```

Comment:
Maybe better to do:
```
                using (var oldOwner = _bufferOwner)
                {
                    _bufferOwner = MemoryPool<byte>.Shared.Rent(Bits.PowerOf2(pos + addedSize));
                    oldOwner.Memory.CopyTo(_bufferOwner.Memory);
                }
```

Reply:
Sure! I'll include it with the unit test soon/tomorrow.

Reply:
Done